20_classes_lab/02.py

```diff
@@ -1,13 +1,17 @@
-"""
-Write a class called MyCounter that counts
-how many times it was initialised, so the
-following code:
+#!/usr/bin/python
+global_c = 1
+class MyCounter:
+  def __init__(self):
+    global global_c
+    self.count = global_c
+    global_c += 1
```

Comment:
This works but you'll get nicer result by using a class variable:

```
class MyCounter:
  instance_count = 0
  
  def __init__(self):
    MyCounter.instance_count += 1
```

That way you can have many classes without fearing name collision in global variable names